Update recipe, tests, and maintainers

duncanmmacleod opened the pull request:
This PR bundles #4 with updates to the recipe syntax, and addition of the following maintainers:

- @duncanmmacleod (me)
- @erinelizabethseder-virgo

Closes #4 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

regro-cf-autotick-bot committed:
update to CFEP-25 `noarch: python` syntax

conda-forge-webservices[bot] committed:
[ci skip] added user duncanmmacleod

duncanmmacleod committed:
Add @erinelizabethseder-virgo as maintainer

duncanmmacleod committed:
Recipe cleanup

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

duncanmmacleod committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.04.13.15.23

Other tools:
- conda-build 25.9.0
- rattler-build 0.48.1
- rattler-build-conda-compat 1.4.9
